api/tests/acceptance/application/assessment-controller-get-adaptive_test.js

```diff
@@ -120,21 +120,14 @@ describe('Acceptance | API | assessment-controller-get-adaptive', function() {
       return knex('assessments').delete();
     });
 
-    it('should return HTTP status code 204 when there is not next challenge', (done) => {
+    it('should return HTTP status code 204 when there is not next challenge', () => {
```

Comment:
'should return HTTP status code 404 when there is not next challenge'

Reply:
Bien vu !